lib/src/gutter.dart

```diff
@@ -1,54 +1,123 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
 
-import 'package:flutter_gutter/flutter_gutter.dart';
-
-/// A gap twice the standard gutter size according to Material Design's
-/// breakpoints system.
-class GutterLarge extends StatelessWidget {
-  const GutterLarge({super.key});
-
-  @override
-  Widget build(BuildContext context) => Gap(size: context.gutterLarge);
-}
+import '../flutter_gutter.dart';
 
 /// A gap of the standard gutter size according to Material Design's breakpoints
 /// system.
+/// Creates a `SizedBox` with a height of size corresponding to the layout axis
+/// of the first `Scrollable`, `Row` or `Column` above this widget.
 class Gutter extends StatelessWidget {
-  const Gutter({super.key});
+  /// Creates a new [Gutter] widget.
+  const Gutter({
+    super.key,
+    this.size,
+    this.type = GutterType.medium,
+    this.scaleFactor,
```

Comment:
Can be left for a later PR (in fact I can do it myself) but I've been for awhile thinking about adding a `*` operator overload that would mutate and multiply against `scaleFactor` so you could do stuff like:
`Gutter() * 3`
as syntactic sugar for a Gutter with a scaleFactor of 3.

Reply:
Sounds good.